plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/InMemoryGlueCache.java

```diff
@@ -87,7 +87,7 @@ public InMemoryGlueCache(Duration metadataCacheTtl, Duration columnStatisticsCac
         this.allFunctionsCache = buildCache(metadataCacheTtlMillis, maximumSize, ValueHolder::new);
         this.functionCache = buildCache(metadataCacheTtlMillis, maximumSize, ValueHolder::new);
 
-        OptionalLong columnStatisticsCacheTtlMillis = OptionalLong.of(metadataCacheTtl.toMillis());
+        OptionalLong columnStatisticsCacheTtlMillis = OptionalLong.of(columnStatisticsCacheTtl.toMillis());
```

Comment:
Variable name should be `statsCacheTtl` instead of `columnStatisticsCacheTtl`

Reply:
thanks, fixed